cuesubmit/plugins/blender/OpenCue-Blender/__init__.py

```diff
@@ -0,0 +1,200 @@
+#  Copyright Contributors to the OpenCue Project
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+
+bl_info = {
+    "name": "OpenCue",
+    "author": "Nuwan Jayawardene",
```

Comment:
Please avoid authors on the source file. If a author field is require, please use `Academy Software Foundation`